docs/api/transaction/get-raw-transaction.example.json

```diff
@@ -0,0 +1,3 @@
+{ 
+  "raw_tx": "test-raw-tx" 
```

Comment:
`"raw_tx: "0x1234"` would be more inline with the other examples

Reply:
done.